tests/storage/mpathconf_test.py

```diff
@@ -224,3 +224,157 @@ def test_mpathd_parser():
             Section('entry', [])
         ]),
     ]
+
+
+def test_mpathd_ufn_enabled_single():
+    fake_conf = [
+        Section('default', [
```

Comment:
Just for consistency, this is called "defaults". If you send another version it would be nice to rename in all tests.